src/main.cpp

```diff
@@ -80,35 +80,63 @@ StartSubCommand CheckStartSubCommands(const wchar_t* arg)
 
 
 int ParseStopOrAnalyze(int argc, wchar_t* argv[], const wchar_t* command, const wchar_t* sessionOrInputHelp,
-    std::wstring& firstArg, std::wstring& outputFile, bool& analyzeTemplates)
+    std::wstring& firstArg, std::wstring& outputFile, bool& analyzeTemplates, bool& generateTimeTrace)
 {
     if (argc < 4)
     {
-        std::wcout << L"vcperf.exe " << command << " [/templates] " << sessionOrInputHelp << " outputFile.etl" << std::endl;
+        std::wcout << L"vcperf.exe " << command << " [/templates] " << sessionOrInputHelp << " [/timetrace] outputFile.etl" << std::endl;
```

Comment:
I think it would look better if you display this on two lines. One for the WPA command and another for the /timetrace option.

Reply:
Sure, I think it will be clearer.
Similar to the update on the `README.md`.

Thanks for the suggestion!